plugins/ftdidmx/FtdiDmxPlugin.cpp

```diff
@@ -44,14 +49,6 @@ const char FtdiDmxPlugin::PLUGIN_PREFIX[] = "ftdidmx";
  * Ownership of the FtdiDmxDevice is transfered to us here.
  */
 void FtdiDmxPlugin::AddDevice(FtdiDmxDevice *device) {
-  // Check if device is working before adding
```

Comment:
Since a device can have one or more (in/)output ports the test is done when before calling AddPort(), so if the test fails the port never gets added, for a single port device this would mean that OLA knows about it's existence but can't add a universe to it, for a multiple port device this could mean the same (if all ports failed) or it could mean that it can provide less universes then it physically has connections for....